apps/www/pages/_app.tsx

```diff
@@ -2,53 +2,47 @@ import '@code-hike/mdx/styles'
 import 'config/code-hike.scss'
 import '../styles/index.css'
 
+import { useEffect } from 'react'
 import { SessionContextProvider } from '@supabase/auth-helpers-react'
-import { AuthProvider, ThemeProvider, useTelemetryProps, useThemeSandbox } from 'common'
-import { DefaultSeo } from 'next-seo'
 import { AppProps } from 'next/app'
 import Head from 'next/head'
 import { useRouter } from 'next/router'
-import { useEffect } from 'react'
+import { DefaultSeo } from 'next-seo'
+
 import { PortalToast, themes } from 'ui'
 import { CommandProvider } from 'ui-patterns/CommandMenu'
 import { useConsent } from 'ui-patterns/ConsentToast'
-
+import Telemetry from '~/lib/telemetry'
+import supabase from '~/lib/supabase'
+import { AuthProvider, ThemeProvider, useTelemetryProps, useThemeSandbox } from 'common'
 import MetaFaviconsPagesRouter, {
   DEFAULT_FAVICON_ROUTE,
   DEFAULT_FAVICON_THEME_COLOR,
 } from 'common/MetaFavicons/pages-router'
-import { WwwCommandMenu } from '~/components/CommandMenu'
-import { API_URL, APP_NAME, DEFAULT_META_DESCRIPTION } from '~/lib/constants'
-import { post } from '~/lib/fetchWrapper'
-import supabase from '~/lib/supabase'
 import useDarkLaunchWeeks from '../hooks/useDarkLaunchWeeks'
+import { APP_NAME, DEFAULT_META_DESCRIPTION } from '~/lib/constants'
+import { WwwCommandMenu } from '~/components/CommandMenu'
 
 export default function App({ Component, pageProps }: AppProps) {
   const router = useRouter()
   const telemetryProps = useTelemetryProps()
-  const { consentValue, hasAcceptedConsent } = useConsent()
+  const { consentValue } = useConsent()
 
   useThemeSandbox()
 
-  function handlePageTelemetry(route: string) {
-    return post(`${API_URL}/telemetry/page`, {
-      referrer: document.referrer,
-      title: document.title,
-      route,
-      ga: {
-        screen_resolution: telemetryProps?.screenResolution,
-        language: telemetryProps?.language,
+  function handleRouteChange(url: string) {
+    Telemetry.sendEvent(
+      {
+        category: 'www',
+        action: 'page_views',
+        label: url,
       },
-    })
+      telemetryProps,
+      router
+    )
   }
```

Comment:
why converting from `telemetry/page` to `/telemetry/event`?

Reply:
on studio, we still use `/telemetry/page` https://github.com/supabase/supabase/blob/cd55df330df8d9415c7ff183aa92538c14c32e06/apps/studio/components/ui/PageTelemetry.tsx#L80-L90

Reply:
`/telemetry/page` and `/telemetry/event` are different. I don't think we can create a page_view event to replace the call to `/telemetry/page`."

Reply:
Ah you're right. I'm gonna revert this "refactoring" on the `www/_app` then.